buildSrc/build.gradle

```diff
@@ -260,10 +260,7 @@ if (project != rootProject) {
    */
   afterEvaluate {
     generatePomFileForPluginMavenPublication.enabled = false
-    tasks.named("validatePluginMavenPom").configure {
-      dependsOn("generatePomFileForNebulaPublication")
-    }
-    tasks.named("publishPluginMavenPublicationToMavenLocal").configure {
+    tasks.matching { it.name.contains("PluginMaven") }*.configure {
```

Comment:
@dblock more robust fix comparing to https://github.com/opensearch-project/OpenSearch/pull/1624 to cover all plugin publishing tasks